tests/vectors/resolution/afterDeactivate.json

```diff
@@ -0,0 +1,3 @@
+{
+  "status": "deactivated"
```

Comment:
I wasn't able to find this field in the spec.

I think we only currently note that the HTTP status is 410.

Reply:
@troyronda, please file a separate bug if you think this implementation must not add anything in boy in deactivate case. But this is the existing behavior so should not block the PR IMO.

Reply:
@thehenrytsai  I don't want to block the PR due to existing behavior.